dev/Common/ColorConversion.cpp

```diff
@@ -246,12 +249,7 @@ Rgb HsvToRgb(const Hsv &hsv)
 
 Rgb HexToRgb(const wstring_view& input)
 {
-    Rgb rgbValue;
-    double ignored;
-
-    HexToRgba(input, &rgbValue, &ignored);
-
-    return rgbValue;
+    return HexToRgba(input).rgb;
 }
```

Comment:
if all the callers want a std::tuple<Rgb, double> then maybe this function should return that type too?

Reply:
I'll leave it as it is because when calling this method, the caller seems to want to set alpha to whatever it wants.